src/Rules/ConsoleCommand/NoConstructorDependencyInjectionRule.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Larastan\Larastan\Rules\ConsoleCommand;
+
+use Illuminate\Console\Command;
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Node\InClassMethodNode;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleError;
+use PHPStan\Rules\RuleErrorBuilder;
+
+use function count;
+use function sprintf;
+
+/** @implements Rule<InClassMethodNode> */
+final class NoConstructorDependencyInjectionRule implements Rule
+{
+    public function getNodeType(): string
+    {
+        return InClassMethodNode::class;
+    }
+
+    /**
+     * @param InClassMethodNode $node
+     *
+     * @return RuleError[] errors
+     */
+    public function processNode(Node $node, Scope $scope): array
+    {
+        $method = $node->getMethodReflection();
+
+        if (! $method->isConstructor()) {
+            return [];
+        }
+
+        $classReflection = $node->getClassReflection();
+
+        if (! $classReflection->isSubclassOf(Command::class)) {
```

Comment:
`isSubclassOf` with a class string is deprecated, this should be:


```suggestion
        if (! $classReflection->is(Command::class)) {
```